docs/fragments.rst

```diff
@@ -0,0 +1,19 @@
+Fragments
+=========
+
+Fragment description here...
+
+Fragment Directories
+--------------------
+
+To include new fragments into a Castle site using a non-theme add-on, simply
```

Comment:
OK, pita mode: it's best to avoid using words like 'simply' or 'just', and do you mean a directory in the theme?

Reply:
Ah, a directory in the add-on you're developing. Would be ideal to have an example directory name, where it should be relatively inside the package, and even use a real fragment and its name and the directory name.